apps/user/src/app/form/전형선택/전형선택.tsx

```diff
@@ -21,36 +20,36 @@ const 전형선택 = ({ onPrevious, onNext }: PropsType) => {
                     label="입학 전형 선택"
                     name="입학전형선택"
                     list={['일반전형', '특별전형']}
```

Comment:
마루 개발 서버에 원서 Form에 type 속성의 타입에 맞는 값을 줘야하니 list 프롭 `{ value: string, content: string }[]` 타입으로 넘겨야 하지 않을까요..?
```suggestion
                    list={[
                        { value: 'REGULAR', content: '입학전형' },
                        { value: 'MEISTER_TALENT', content: '특별전형' }
                    ]}
```
이렇게요